projects/lib/ng-package.json

```diff
@@ -2,11 +2,7 @@
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/lib",
   "lib": {
-    "entryFile": "src/public_api.ts",
-    "umdModuleIds": {
```

Comment:
Do these need to be deleted or was this part of trying to get the tests working?

Reply:
This was automatically removed by Angular CLI migration to version 13.

Reply:
Shall I therefore resolve this conversation? Or is the author of conversation resolving it?

Reply:
Shall I therefore resolve this conversation? Or is the author of conversation resolving it?

Reply:
I'll resolve it :+1: